src/legacy/core_plugins/kibana/public/home/components/home.js

```diff
@@ -247,6 +251,10 @@ export class Home extends Component {
 
 Home.propTypes = {
   addBasePath: PropTypes.func.isRequired,
+  fetchTelemetry: PropTypes.func.isRequired,
+  getTelemetryBannerId: PropTypes.func.isRequired,
+  setOptIn: PropTypes.func.isRequired,
+  showTelemetryOptIn: PropTypes.bool.isRequired,
```

Comment:
it was a little hard to determine that this variable is a boolean, without reading down to here. Maybe `shouldShowTelemetryOptIn`?